templates/cook/cook_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <div class="container my-5">
+    <div class="d-flex justify-content-between align-items-center mb-4">
+      <h1>Cooks</h1>
+      <a href="{% url 'management_system:cook-create' %}" class="btn btn-primary">Create Cook</a>
+    </div>
+
+    {% if cook_list %}
+      <div class="table-responsive">
+        <table class="table table-striped table-hover">
+          <thead>
+          <tr>
+            <th scope="col">#</th>
+            <th scope="col">First Name</th>
+            <th scope="col">Last Name</th>
+            <th scope="col">Details</th>
+          </tr>
+          </thead>
+          <tbody>
+          {% for cook in cook_list %}
+            <tr>
```

Comment:
also add a column for ordinal number, so user see how many records in the table (starting by 1)

Reply:
Thanks, I will implement this feature, a little bit later on.